lib/paper_trail/reifiers/belongs_to.rb

```diff
@@ -39,7 +39,7 @@ def load_record(assoc, id, options, version)
         # @api private
         def load_version(assoc, id, transaction_id, version_at)
           assoc.klass.paper_trail.version_class.
-            where("item_type = ?", assoc.klass.name).
+            where("item_type = ?", assoc.klass.base_class.name).
```

Comment:
Nice. I was hoping AR provided something like this `base_class`, but I couldn't find it.